### DISCUSSION TITLE
Sketch

### DISCUSSION THREAD

**Discussion by crazyweeds** [DouyinFE/semi-design#968] [Announcements]
不知道会不会提供类似Ant Design Pro类似的Sketch资源下载呢？

**Comment by pointhalo** [comment]
关于 Sketch ，我们内部进行过讨论，结论是【不计划提供】

主要基于以下几点考虑：

1、Semi 团队一直在尝试做更多设计链路与研发链路的结合，包括Semi DSM、Semi 设计插件，Code2Design 等我们都在积极推进，这类工作会与Figma强绑定，sketch无法满足我们的本身需求。

2、作为一个灵活的Design System，过去两年 Semi 在我们公司内部也一直在跟随业务需求在进行代码高速迭代，现在开源后也一样，伴随着的是设计样式、交互的持续进化。  
由于一些人力方面的原因，以及我们团队自身情况（我们团队本身是Figma的重度用户，Sketch本身并不在我们的工作流当中）我们没有资源去单独再搞一版sketch的设计资源并长期迭代维护。  
如果设计资产长期与代码资产脱节，也会加重研发使用的成本，我们认为并不是一个好的选择。

3、也尝试过一些低成本适配的方案（例如某些figma转sketch的开源插件），转换效果不是很理想。
